db/schema.rb

```diff
@@ -0,0 +1,53 @@
+# This file is auto-generated from the current state of the database. Instead
+# of editing this file, please use the migrations feature of Active Record to
+# incrementally modify your database, and then regenerate this schema definition.
+#
+# This file is the source Rails uses to define your schema when running `bin/rails
+# db:schema:load`. When creating a new database, `bin/rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema[7.0].define(version: 2022_11_22_192817) do
+  # These are extensions that must be enabled in order to support this database
+  enable_extension "plpgsql"
+
+  create_table "groups", force: :cascade do |t|
+    t.string "name"
+    t.string "icon"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.bigint "user_id", null: false
+    t.index ["user_id"], name: "index_groups_on_user_id"
+  end
+
+  create_table "spendings", force: :cascade do |t|
+    t.string "name"
+    t.integer "amount"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.bigint "group_id", null: false
+    t.bigint "user_id", null: false
```

Comment:
* do not change column names visible in the diagram check project [requirement for reference](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=do%20not%20change%20column%20names%20visible%20in%20the%20diagram.%20You%20need%20to%20change%20the%20%22Entity%22%20name%20according%20to%20the%20theme%20you%20have%20chosen%20for%20your%20project%20(please%20note%3A%20%22Transaction%22%20is%20a%20name%20already%20used%20by%20ActiveRecord%2C%20so%20using%20it%20as%20a%20name%20for%20your%20model%20and%20table%20will%20result%20in%20an%20error).) . Kindly stick to the column names as seen in the ERD diagram 🙏